winui/Rating/Customization.md

```diff
@@ -9,4 +9,242 @@ documentation: ug
 
 # Customization in WinUI Rating
 
-This section explains the customization features available in the WinUI Rating control.
+This section explains the customization features available in the WinUI `Rating` control.
+
+## Style
+
+The Rating control style can be customized using the `RatedItemStyle`, `UnratedItemStyle` Property.
+
```

Comment:
@Priyanka-Vijayasankar The Rating control style can be customized by using the `RatedItemStyle` and `UnratedItemStyle` properties.

Reply:
Changed the style content.